psse/psse-model/src/main/java/com/powsybl/psse/model/pf/PsseTransformer.java

```diff
@@ -0,0 +1,406 @@
+        ownership.setModel(model);
+        winding1.setModel(model);
+        winding2.setModel(model);
+        winding3.setModel(model);
+        winding1Rates.setModel(model);
+        winding2Rates.setModel(model);
+        winding3Rates.setModel(model);
+    }
+
+    @Parsed(field = {"i", "ibus"})
+    private int i;
+
+    @Parsed(field = {"j", "jbus"})
+    private int j;
+
+    @Parsed(field = {"k", "kbus"})
+    private int k = 0;
+
+    @Parsed(defaultNullRead = "1")
+    private String ckt;
+
+    @Parsed
+    private int cw = 1;
+
+    @Parsed
+    private int cz = 1;
+
+    @Parsed
+    private int cm = 1;
+
+    @Parsed
+    private double mag1 = 0;
+
+    @Parsed
+    private double mag2 = 0;
+
+    @Parsed(field = {"nmetr", "nmet"})
+    private int nmetr = 2;
+
+    @Parsed(defaultNullRead = "            ")
+    private String name;
+
+    @Parsed
+    private int stat = 1;
+
+    @Nested
+    private PsseOwnership ownership;
+
+    @Parsed(defaultNullRead = "            ")
+    // If the issue 432 in Univocity is fixed,
+    // the previous annotation will be correctly processed
+    // and there would be no need to initialize vecgrp with default value
+    // (https://github.com/uniVocity/univocity-parsers/issues/432)
+    private String vecgrp = "            ";
+
+    @Parsed
+    @Revision(since = 35)
+    private int zcod = 0;
+
+    @Parsed(field = {"r12", "r1_2"})
+    private double r12 = 0;
+
+    @Parsed(field = {"x12", "x1_2"})
+    private double x12;
+
+    @Parsed(field = {"sbase12", "sbase1_2"})
+    private double sbase12 = Double.NaN;
+
+    @Parsed(field = {"r23", "r2_3"})
+    private double r23 = 0;
+
+    @Parsed(field = {"x23", "x2_3"})
+    private double x23 = Double.NaN;
+
+    @Parsed(field = {"sbase23", "sbase2_3"})
+    private double sbase23 = Double.NaN;
+
+    @Parsed(field = {"r31", "r3_1"})
+    private double r31 = 0;
+
+    @Parsed(field = {"x31", "x3_1"})
+    private double x31 = Double.NaN;
+
+    @Parsed(field = {"sbase31", "sbase3_1"})
+    private double sbase31 = Double.NaN;
+
+    @Parsed
+    private double vmstar = 1;
+
+    @Parsed
+    private double anstar = 0;
+
+    @Nested(headerTransformer = WindingHeaderTransformer.class, args = "1")
+    private PsseTransformerWinding winding1;
+
+    @Nested(headerTransformer = WindingRatesHeaderTransformer.class, args = "1")
+    private PsseRates winding1Rates;
+
+    @Nested(headerTransformer = WindingHeaderTransformer.class, args = "2")
+    private PsseTransformerWinding winding2;
+
+    @Nested(headerTransformer = WindingRatesHeaderTransformer.class, args = "2")
+    private PsseRates winding2Rates;
+
+    @Nested(headerTransformer = WindingHeaderTransformer.class, args = "3")
+    private PsseTransformerWinding winding3;
+
+    @Nested(headerTransformer = WindingRatesHeaderTransformer.class, args = "3")
+    private PsseRates winding3Rates;
+
+    public static class WindingHeaderTransformer extends HeaderTransformer {
```

Comment:
We should extract `WindingHeaderTransformer` and `WindingRatesHeaderTransformer` from this class because they are related to I/O

Reply:
Moved